scripts/deploy_oak_functions_loader

```diff
@@ -27,6 +26,9 @@ docker push "${FUNCTIONS_DOCKER_IMAGE_NAME}:latest"
 
 readonly FUNCTIONS_EXAMPLE_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-functions-weather-lookup'
 
+# Copy the certificate file to the example directory.
+cp ./examples/certs/local/ca.pem ./oak_functions/examples/weather_lookup
```

Comment:
What is this cert for?

Reply:
The tee_certificate, required by the config file: https://github.com/project-oak/oak/blob/86e1ef35564f1e3f431810121fa3bbb2fa3fdc37/oak_functions/examples/weather_lookup/config.toml#L4

Reply:
Ah, right, I keep forgetting that, it still does not seem very natural to me. It's just a binary blob that gets sent by the server at some point in the remote attestation protocol. In the same way, we will have to also have an inclusion proof in a verifiable log of the hash, and other similar things. I think it should be renamed to something more generic, and also not pass another unrelated certificate. Cc @ipetr0v

Reply:
What should be passed instead of this certificate? At the moment it looks like that the attestation protocol ignores it. So, I could remove this and use an empty vector. But if we have to redo this later, I'll keep it like this. 
On the other hand, I think we should make `tee_certificate_path` a command line option, instead of including it in the config. It would make it easier to use different files in different paths for CI builds, and cloud run deployments. WDYT?

Reply:
I think it's fine to leave it in for now, but I do think we need to clarify the story, the current structure is not particularly realistic. Even if we wanted to provide the AMD cert there, why would the server send it to the client? What would the client verify about it? What stops the server from sending a self-signed cert? Or is the intention that the client has the AMD root cert already, and this is just one or more intermediate certs that are machine-specific? And even if they are machine-specific, surely they would not appear as files in a Docker image.

Reply:
IIUC provider's certificate should be signed by a certificate that client can check (i.e. it's rooted in a well known certificate).

Reply:
But where does it come from? And who puts it there?

Reply:
It should be available by the remote attestation implementation alongside the TEE report.

Reply:
So it's unlikely it will be loaded from a file path, right? By the way, instead, something like a log inclusion proof would actually be passed as a file eventually, so I think this makes more sense for something like that

Reply:
Just to be clear, this is the `tee_certificate` in `AttestationServer` that we are talking about: https://github.com/project-oak/oak/blob/b8a986aefca80d26604d227e2283ad80a7e96006/experimental/grpc_attestation/src/attestation.rs#L115-L120. Is that right?

Reply:
Yes that is my understanding.

Reply:
Removed it for now.